src/abacus/detail/to_json.cpp

```diff
@@ -62,10 +62,26 @@ auto to_json(const view& view, bool slim) -> bourne::json
         }
         else
         {
+            std::string kind;
+            switch (view.kind(i))
+            {
+            case metric_kind::constant:
+                kind = "constant";
+                break;
+            case metric_kind::counter:
+                kind = "counter";
+                break;
+            case metric_kind::gauge:
+                kind = "gauge";
+                break;
+            default:
+                assert(false);
+                break;
+            }
```

Comment:
it's fine, but we could also add a to_string function in the abacus public API which could convert the kind enum (and maybe the others) to a string?